lib/ssz_types/signed_voluntary_exit.ex

```diff
@@ -0,0 +1,19 @@
+defmodule SszTypes.SignedVoluntaryExit do
+  @moduledoc """
+  Struct definition for `SignedVoluntaryExit`.
+  Related definitions in `native/ssz_nif/src/types/`.
+  """
+
+  fields = [
+    :message,
+    :signature
+  ]
+
+  @enforce_keys fields
+  defstruct fields
+
+  @type t :: %__MODULE__{
+          message: SszTypes.VoluntaryExit,
```

Comment:
The struct types are (by convention) the `t` function of the module
```suggestion
          message: SszTypes.VoluntaryExit.t(),
```